src/jbi/whiteboard_actions/default.py

```diff
@@ -69,15 +69,17 @@ def comment_create_or_noop(self, payload: BugzillaWebhookRequest):
             )
             return {"status": "noop"}
 
-        jira_response = self.jira_client.issue_add_comment(
-            issue_key=linked_issue_key,
-            comment=payload.map_as_jira_comment(),
-        )
-        logger.debug(
-            "Comment added to Jira issue %s",
-            linked_issue_key,
-            extra=log_context,
-        )
+        comment = payload.map_as_jira_comment()
+        if comment is not None:
+            jira_response = self.jira_client.issue_add_comment(
+                issue_key=linked_issue_key,
+                comment=payload.map_as_jira_comment(),
+            )
+            logger.debug(
+                "Comment added to Jira issue %s",
+                linked_issue_key,
+                extra=log_context,
+            )
         return {"status": "comment", "jira_response": jira_response}
```

Comment:
If this doesn't crash about `jira_response`, this line is probably not executed

Reply:
I theory it should never happen that we can't get the comment from bugzilla (unless maybe the comment can be deleted between the webhook and the attempt to retrieve?). Maybe it would be better to raise an exception in that case?

Reply:
What I meant is that if comment is None then `jira_response` is not defined